io/fs/ffsb.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 Red Hat, Inc.
+# Flexible Filesystem Benchmark (FFSB) is a cross-platform filesystem
+#  performance measurement tool
+# Ported to Avocado from Kalpana S Shetty(kalshett@linux.vnet.ibm.com)
+# source: https://github.com/autotest/autotest-client-tests/tree/master/ffsb
```

Comment:
This can be improved. The original author seems to be here:
https://github.com/autotest/autotest-client-tests/blob/master/ffsb/control
Also, the copyright it not Red Hat for 2016, but IBM, since you are the one porting it. And before that, it's also IBM, since the original author's e-mail is from IBM as well. Here an example:
https://github.com/avocado-framework/avocado-misc-tests/blob/master/generic/error_cleanup.py